wp-includes/widgets/class-wp-widget-audio.php

```diff
@@ -23,9 +23,10 @@ class WP_Widget_Audio extends WP_Widget_Media {
 	 * @access public
 	 */
 	public function __construct() {
-		parent::__construct( 'audio', __( 'Audio' ), array(
-			'classname'   => 'widget_audio',
+		parent::__construct( 'media-audio', __( 'Audio' ), array(
```

Comment:
One more thing…™

Should the ID base be `media_audio` or `media-audio`? Core is not consistent here. For multi-word widget ID bases, there are:

* `nav_menu`
* `recent-comments`
* `recent-posts`
* `tag_cloud`

So should we use `-` or `_`?

If I recall correctly, the REST API favors hyphens so I've inquired about that: https://wordpress.slack.com/archives/core-restapi/p1489088798492976